### DISCUSSION TITLE
UI Customization in the works.

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#3600] [Show and tell]
After several people asked for the possibility to customize the Webfrontend and other aspects in design I have startet a private fork which will later hopefully be pulled back to create a user friendly Customization UI.
https://github.com/Ylianst/MeshCentral/issues/3578

I will try to document progress here and hopefully have a first basic version ready in the next couple of weeks.
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/68963439/152424274-59e374ec-b8db-4dce-b59f-e8645b6225f4.png">

**Comment by JSuenram** [comment]
Small update,
I will code freeze based on 0.9.84 and adapt my Customization UI based on this version.
When everything works as expected I will merge this in my fork and do some more tests.
Starting to migrate CSS to Tailwind, looks promising so far.

**Comment by VINISHVV** [comment]
@JSuenram - Nice. Waiting for these changes.

**Comment by VINISHVV** [comment]
@JSuenram - any progress

**Reply by JSuenram** [reply]
> @JSuenram - any progress

I am currently busy in other projects. But this is still on my list....

**Reply by VINISHVV** [reply]
Hi,

Any hope ?

**Reply by JSuenram** [reply]
I am sorry... still far to busy for even thinking about developing anything at the moment.....
Times are crazy in IT-Business the last months... and still getting more strange every week....

I am really sorry.... wish I could say other wise....

**Comment by JSuenram** [comment]
Quick update here after some time... making small progress about recreating the frontend with more theme options....

Have a sneek preview here: I call it "MeshCentral Next" work in progress an suspected to change....

<img width="1536" height="1024" alt="MCNext" src="https://github.com/user-attachments/assets/1aae6b58-42b5-4ee1-891c-cb02543b19e3" />

**Reply by si458** [reply]
oh cool! this maybe called `default4` the `next extreme theme` haha
